### PR TITLE
Rename To Datatrails

### PR DESCRIPTION
**Pull request by honourfish**
## Overview
* rename all references to DataTrails
* ensure the default fqdn is the correct datatrails one

## Testing

tested using a known receipt:
```
$> datatrails_receipt_scittv1 verify -d <receipt>
```

```
{
  "anchor": "f29b54ec5f183ba17a434809bef8a06e457aa63cd99deab3bcb54715901f0910",
  "api_query": "https://app.datatrails.ai/archivist/v2/assets/-/events?order_by=SIMPLEHASHV2&proof_mechanism=SIMPLE_HASH&timestamp_accepted_since=2023-06-01T13:33:00Z&timestamp_accepted_before=2023-06-08T11:18:02Z",
  "endOperator": "lt",
  "endTimeRFC3339": "2023-06-08T11:18:02Z",
  "endTimeUnix": 1686223082,
  "eventCount": 1,
  "hashSchemaVersion": 2,
  "proofMechanism": 2,
  "startOperator": "gte",
  "startTimeRFC3339": "2023-06-01T13:33:00Z",
  "startTimeUnix": 1685626380,
  "tenant": "tenant/c8c51acf-5084-48ba-9e76-2c3cd9439818"
}
```
[AB#8751](https://dev.azure.com/jitsuin/0629f48c-3979-4bbc-9026-cb06b3dfd0ae/_workitems/edit/8751)

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -81,12 +81,12 @@
...
 # or fully qualified paths (eg. https://...)
-# Note this assumes that the contents of docs/rkvst-css/* are copied to _site/css/*
+# Note this assumes that the contents of docs/datatrails-css/* are copied to _site/css/*
 html_css_files = [
-    "css/rkvst_theme.css",
+    "css/datatrails_theme.css",
 ]
 html_favicon = "_static/favicon.ico"
-html_logo = "_static/RKVST_WhtLogo_RGB.png"
+html_logo = "_static/DataTrails_WhtLogo_RGB.png"
```
Comment (honourfish) [inline docs/conf.py:89]: NOTE: need a new logo here!

Comment (honourfish) [reply docs/conf.py:89]: thanks to Caitlin have a logo!

File: README.rst [hunk 2]
```diff
@@ -1,10 +1,10 @@
 
 .. _readme:
 
-RKVST Scitt Receipt Verification in python
+DataTrails Scitt Receipt Verification in python
 ==========================================
```
Comment (j-hartley) [inline README.rst:5]: do we want to make this line a bit longer to match the title length?

File: datatrails_receipt_scitt/namedproofs.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
 """
 This file deals in a generic way with the name_proofs component of the contents
-defined for the receipt by the RKVST  EIP1186namedProofs Tree algorithm.
+defined for the receipt by the DataTrails EIP1186namedProofs Tree algorithm.
 
-See khipureceipt.py or simplehashreceipt.py for rkvst specific conveniences.
+See khipureceipt.py or simplehashreceipt.py for datatrails specific conveniences.
```
Comment (j-hartley) [inline datatrails_receipt_scitt/namedproofs.py:5]: We should remove all mention of khipu while we are in here if possible please?

File: docs/_static/DataTrails_WhtLogo_RGB.png.png
Comment (j-hartley) [inline docs/_static/DataTrails_WhtLogo_RGB.png.png:1]: why .png.png?

**Review by j-hartley — DISMISSED** [review]

File: docs/_static/DataTrails_WhtLogo_RGB.png.png
Comment (honourfish) [reply docs/_static/DataTrails_WhtLogo_RGB.png.png:1]: haha! good spot, it should only have 1 png

File: datatrails_receipt_scitt/namedproofs.py (hunk 3 above)
Comment (honourfish) [reply datatrails_receipt_scitt/namedproofs.py:5]: Can we keep the khipu removal to a separate PR? I am happy to do it, but i think its a better history if we do datatrails in one, and khipu in another?

File: .github/workflows/publish_package.yaml [hunk 4]
```diff
@@ -41,4 +41,4 @@ jobs:
 #       with:
 #         github_token: ${{ secrets.GITHUB_TOKEN }}
 #         publish_dir: "./docs/_build/html"
-#         cname: python-scitt.rkvst.com
+#         cname: python-scitt.datatrails.com
```
Comment (eccles) [inline .github/workflows/publish_package.yaml:44]: datatrails.ai

File: docs/index.rst [hunk 5]
```diff
@@ -13,7 +13,7 @@
 
    api
 
-Keep an eye on `RKVST Docs <https://docs.rkvst.com/docs/overview/introduction/>`_ For guides and howto's
+Keep an eye on `DataTrails Docs <https://docs.datatrails.com/docs/overview/introduction/>`_ For guides and howto's
```
Comment (eccles) [inline docs/index.rst:16]: datatrails.ai

**Review by eccles — CHANGES_REQUESTED** [review]

File: docs/_static/DataTrails_WhtLogo_RGB.png.png
Comment (honourfish) [reply docs/_static/DataTrails_WhtLogo_RGB.png.png:1]: done

File: README.rst (hunk 2 above)
Comment (honourfish) [reply README.rst]: done!

File: datatrails_receipt_scitt/namedproofs.py (hunk 3 above)
Comment (honourfish) [reply datatrails_receipt_scitt/namedproofs.py:5]: defering to a different PR :)

**Review by eccles — APPROVED** [review]

**Pull request merged**